twitcher/adapter/base.py

```diff
@@ -9,14 +10,24 @@
     from pyramid.request import Request
 
 
-class AdapterInterface(object):
+class AdapterBase(type):
+    @property
+    def name(cls):
+        return '{}.{}'.format(cls.__module__, cls.__name__)
+
+
+class AdapterInterface(six.with_metaclass(AdapterBase)):
```

Comment:
is `six` needed? The rest of twitcher is Python 3.x only.

Reply:
We could use the `abc` abstract class variation. 
I have used this variant because of back compatibility (ie: Magpie still allows Python 2)

Reply:
I'm currently skipping Python 2.7 support everywhere and twitcher might not work on python 2.7 anymore. I leave it to you :) Feel free to merge when ready.

Reply:
That's good, I'm planning to drop 2.7 also.